cpmdetail.py

```diff
@@ -5,29 +5,30 @@
 
 config = configparser.ConfigParser()
 config.read_file(open(r'api-key.txt'))
-api_key = config.get('API Key', 'api_key')
```

Comment:
I think you forgot to do a pull before you made your changes because I removed this code in my last commit. In your main branch, do a pull, then go to your branch and do a "git merge main" to pull those changes into your code. You may end up with merge conflicts which get a bit harder to handle.

Reply:
![image](https://github.com/jmirving/cpm-detail/assets/163681461/0df0e360-8211-46f0-9a4e-83232c90b4e0)
I may have copied them into the item after, but my Main was up to date.